packages/atomic/src/images/arrow-right.svg

```diff
@@ -0,0 +1,3 @@
+<svg viewBox="0 0 7 12" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M1.09619 10.5962L5.69239 6L1.09619 1.40381" stroke="#C5CACF" stroke-linecap="round" stroke-linejoin="round"/>
```

Comment:
Is #C5CACF the real color of the separator?

If so, the contrast against white of it is a little low, and we should probably be using a color from a variable instead.